pkg/rtcp/transport_layer_nack.go

```diff
@@ -0,0 +1,114 @@
+package rtcp
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+)
+
+// The TransportLayerNack packet informs the encoder about the loss of a transport packet
+type TransportLayerNack struct {
```

Comment:
Can you link to the spec or copy some of the diagrams here? It would be convenient to have while reading.